### ISSUE TITLE
revisit modulepreload options?

### ISSUE THREAD

**Issue by benmccann** [sveltejs/kit#11190]
### Describe the problem

safari and firefox have support now. do we need the other options still? safari and firefox are rolling out support and being adopted quickly

some of the other options result in double requests. see https://github.com/svelte-society/summit-website/pull/3

### Describe the proposed solution

discuss with team

### Importance

nice to have

**Comment by benmccann** [comment]
https://gs.statcounter.com/browser-version-market-share says most people are currently on Safari 17 using data from December

caniuse is using statcounter data from November. statcounter is currently down, so I can't see if their December data is different from their November data

**Comment by dummdidumm** [comment]
I don't think we should outright remove the option just yet, maybe in SvelteKit 3 or 4. What do you mean about changing the default? The default is `modulepreload` already, which AFAIK is the best option.

**Comment by Rich-Harris** [comment]
In that case I think it makes sense to keep the options for now — they still have value (at least in theory; not sure how many people are using them in the wild) and they really don't add much complexity. I suspect Safari 17 adoption will be high enough by the time we ship SvelteKit 3.0 to remove them then
